src/Organizations/OrganizationEdit/OrganizationEdit.js

```diff
@@ -0,0 +1,100 @@
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { FormattedMessage } from 'react-intl';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+import {
+  Paneset,
+} from '@folio/stripes/components';
+import {
+  LoadingPane,
+} from '@folio/stripes-acq-components';
+
+import FormatTime from '../../Utils/FormatTime';
+import { organizationResourceByUrl } from '../../common/resources';
+import { OrganizationForm } from '../OrganizationForm';
+
+const OrganizationEdit = ({ match, history, location, mutator, stripes }) => {
+  const organizationId = match.params.id;
+
+  const [organization, setOrganization] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(
+    () => {
+      mutator.editOrganizationOrg.GET()
+        .then(organizationsResponse => setOrganization(organizationsResponse))
+        .finally(() => setIsLoading(false));
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
```

Comment:
what if `match.params.id` is changed?

Reply:
component is reloaded when url is changed, so did mount is fine here